plugins/search-backend-node/alpha-api-report.md

```diff
@@ -0,0 +1,62 @@
+## API Report File for "@backstage/plugin-search-backend-node"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { DocumentCollatorFactory } from '@backstage/plugin-search-common';
+import { DocumentDecoratorFactory } from '@backstage/plugin-search-common';
+import { DocumentTypeInfo } from '@backstage/plugin-search-common';
+import { ExtensionPoint } from '@backstage/backend-plugin-api';
+import { SearchEngine } from '@backstage/plugin-search-common';
+import { ServiceRef } from '@backstage/backend-plugin-api';
+import { TaskRunner } from '@backstage/backend-tasks';
+
+// @alpha
+export type IndexServiceBuildOptions = {
```

Comment:
Thinking this should be `SearchIndexServiceStartOptions` with this pattern

Reply:
good catch - will update!